sklearnex/covariance/incremental_covariance.py

```diff
@@ -196,6 +198,45 @@ def _onedal_partial_fit(self, X, queue=None, check_input=True):
 
         return self
 
+    @wrap_output_data
+    def score(self, X_test, y=None):
```

Comment:
I don't really understand mixing `wrap_output_data` and using `get_namespace`? could you please clarify?